app/assets/stylesheets/organizations.scss

```diff
@@ -1,3 +1,10 @@
 // Place all the styles related to the Organizations controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: https://sass-lang.com/
+.form-control {
+  width: inherit;s
```

Comment:
typo here? 😅

Reply:
Oops! In my attempt to push a change that YOU could accept or reject 😓 I inadvertently pushed my change to your branch. I'm sorry. I did it because it was a tiny typo that you may not even have introduced. I'll go ahead merge 🙏🏾